scylla/Cargo.toml

```diff
@@ -34,7 +27,7 @@ bytes = "1.0.1"
 futures = "0.3.6"
 histogram = "0.6.9"
 num_enum = "0.5"
-tokio = { version = "1.12", features = ["net", "time", "io-util", "sync", "rt", "macros"] }
+tokio = { version = "1.28", features = ["net", "time", "io-util", "sync", "rt", "macros"] }
```

Comment:
What was the tokio version bump needed for? The code seems to compile without that.

Reply:
Never mind, of course I changed it to `version = "1.12"` here and Cargo resolved it to version `1.28.2` anyway...

Testing it now again, it looks like at least version `1.27` is required.

Reply:
I guess that a _minor_ version bump of a dependency that appears in the public interface is not considered to be a breaking change, although I wish the PR were more explicit about it in the commit message/PR description.

Let's change it to the lowest version that supports operating on sockets with the `socket2` crate (`1.27`?). The library shouldn't impose more constraints on the dependencies than necessary.